src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -141,6 +160,21 @@ export const NavBar: React.FC<NavBarPropsType> = ({ hasBackButton = true }) => {
 
   const [isRadioPlaying, setIsRadioPlaying] = useState(false);
   const { volume, setVolume } = useRadio(isRadioPlaying, sound);
+  const handleMute = useCallback(
+    (volume: number) => (volume !== 0 ? 0 : volume),
```

Comment:
I don't know if I follow this correctly
```ts
volume !== 0 ? 0 : volume
```

when `volume = 0`, then `volume !== 0` is `false`, so the return is `0`?

I think it may be a scope shadow problem, probably renaming the internal variable to `value` / `v` or something else generic.

Reply:
had to revert back to `: 100` since there's no volume scroll , 433dfe8